project-base/kubernetes/deployments/webserver-php-fpm.yml

```diff
@@ -42,40 +44,40 @@ spec:
                                 path: default.conf
             initContainers:
                 -   name: copy-source-codes-to-volume
-                    image: ~
+                    image: "{{DOCKER_PHP_FPM_IMAGE}}"
                     securityContext:
                         runAsUser: 33
                     command: ["sh", "-c", "cp -r /var/www/html/. /tmp/source-codes"]
                     volumeMounts:
                         -   name: source-codes
                             mountPath: /tmp/source-codes
                         -   name: domains-urls
-                            mountPath: /var/www/html/app/config/domains_urls.yml
+                            mountPath: "{{PATH_CONFIG_DOMAIN_URL}}"
```

Comment:
Why did you parameterize the paths?

Reply:
It was also replaced by `yq` but from your comment I guess it was not necessary.

Reply:
I have noticed that replacement was different (`/var/www/html/project-base/app/config/domains_urls.yml`). After discussion we decided to parametrize only directory path.

Reply:
Makes sense. In that case, I would parameterize just the path to `app/config`.